### DISCUSSION TITLE
Firefox Nightly 108.0a1 (2022-11-08) (64-bit) Address bar not working when TabMixPlus enabled.

### DISCUSSION THREAD

**Discussion by Shahius** [onemen/TabMixPlus#180] [Q&A] [answered]
Dear Onemen,
Nightly just updated to 108.0a1 (2022-11-08) (64-bit), after this update when I type address into address bar and hit Enter, nothing happens.
When I disable TabMixPlus, everything works as usual, address bar works.
I have  1.0.0-pre.11.20221013.1.xpi  installed.
I just tried to install latest dev build, same story, address bar not working when TMP enabled.

They broke something. Do you have any ideas?

**Comment by onemen** [comment]
Thank you for the report @Shahius.

After updating Firefox to the latest Nightly build this morning, I can confirmed that due to changes in the latest Firefox version the address bar does not work.

I see this error message in the Browser Console
```
Tabmix ERROR at TMP_navToolbox_initializeURLBar:
TMP_eventListener.onContentLoaded failed
can't access property "addEventListener", gURLBar.view._rows is undefined
Stack Trace:
  File "chrome://tabmixplus/content/minit/minit.js", line 937, col 7, in TMP_navToolbox_initializeURLBar
  File "chrome://tabmixplus/content/tabmix.js", line 354, col 23, in TMP_EL_onContentLoaded
  File "chrome://tabmixplus/content/tabmix.js", line 1147, col 29, in tabmix_initialization_run
  File "data:application/javascript,
    %2F%2F with bootstrap.js we get here after DOMContentLoaded was fired
    %2F%2F we can call our initializer with 'onContentLoaded' flag
    Tabmix.initialization.init.initialized %3D false%3B
    TMP_eventListener.init()%3B
    Tabmix.initialization.run("onContentLoaded"%2C gBrowser.tabContainer)%3B
  ", line 6, col 27
  File "chrome://tabmix-resource/content/bootstrap/Overlays.jsm", line 552, col 31, in loadScript
  File "chrome://tabmix-resource/content/bootstrap/Overlays.jsm", line 169, col 38, in load
[tabmix.js:1152:18](chrome://tabmixplus/content/tabmix.js)
```

Let me know if you see other messages.

I am working on fixing this issue.

**Comment by onemen** [comment]
[Bug 1798917](https://bugzilla.mozilla.org/show_bug.cgi?id=1798917) - Convert pseudo-private UrlbarView properties and methods to be truly private.
landed on 2022-11-08 16:57

[Bug 1798880](https://bugzilla.mozilla.org/show_bug.cgi?id=1798880) - Mark UrlbarView::getClosestSelectableElement as private.
landed on 2022-11-08 16:37

**Comment by onemen** [comment, marked as answer]
Download and install this [test build](https://tabmixplus.org/dev-builds/test%20builds/1.0.0-pre.11.20221109.1.xpi)

Report back.

**Reply by Shahius** [reply]
Yesss! Address bar is working now with your test build!!! Thank you! If I see some bugs, I will let you know here.
